include/sys/dbuf.h

```diff
@@ -206,6 +206,12 @@ typedef struct dmu_buf_impl {
 	 */
 	struct dmu_buf_impl *db_hash_next;
 
+	/*
+	 * Our link on the owner dnodes's dn_dbufs list.
+	 * Protected by its dn_dbufs_mtx.
+	 */
+	avl_node_t db_link;
+
```

Comment:
I think we should add a comment around here stating which members are intended to be on the same cache line.  Otherwise someone may inadvertently undo this performance improvement.

Reply:
Done